test/dispersion.jl

```diff
@@ -0,0 +1,50 @@
+@testset "Dispersion methods" begin
+    x = rand(100)
+
+    @testset "Dispersion" begin
+        est = Dispersion()
+
+        @testset "Internals" begin
+            # Li et al. (2018) recommends using at least 1000 data points when estimating
+            # dispersion entropy.
+            x = rand(1000)
+            n_categories = 4
+            m = 4
+            τ = 1
+            s = GaussianSymbolization(n_categories = n_categories)
+
+            # Symbols should be in the set [1, 2, …, n_categories].
+            symbols = Entropies.symbolize(x, s)
+            @test all([s ∈ collect(1:n_categories) for s in symbols])
+
+            # Dispersion patterns should have a normalized histogram that sums to 1.0.
+            dispersion_patterns = DelayEmbeddings.embed(symbols, m, τ)
+            hist = Entropies.dispersion_histogram(dispersion_patterns, length(x), m, τ)
+            @test sum(hist) ≈ 1.0
+        end
+
+        ps = probabilities(x, est)
+        @test ps isa Probabilities
+
+        de = entropy_renyi(x, est, q = 1, base = 2, normalize = false)
+        de_norm = entropy_renyi(x, est, q = 1, base = 2, normalize = true)
+        @test de isa Real
+        @test de_norm isa Real
+        @test de >= 0.0
+        @test de_norm >= 0.0
+    end
+
+    @testset "ReverseDispersion" begin
+        est = ReverseDispersion()
+        ps = probabilities(x, est)
+        @test ps isa Probabilities
+
+        de = entropy_renyi(x, est, q = 1, base = 2, normalize = false)
+        de_norm = entropy_renyi(x, est, q = 1, base = 2, normalize = true)
+        @test de isa Real
+        @test de_norm isa Real
+        @test de >= 0.0
+        @test de_norm >= 0.0
```

Comment:
Can we devise any analytically resolved / meanigful tests for this block? We might as well do proper tests in new PRs. 

`de isa Real` is an interface test. It doesn't test the `ReverseDispersion()` method, it tests the `renyi_entropy` interface. Same with `ps isa Probabilities`. We would need tests that actually test whether the algorithm does as it should by devising a scenario where we know what the output should be a priori.

Reply:
Good points. For the normalized case, a meaningful test is that the resulting entropy value is on in the interval `[0,1]`. 

It is also possible to construct analytical test cases for this method, but I propose we do so in a different PR.